### PR TITLE
Adds classnames to links. Mainly for GTM purposes

### PR DESCRIPTION
**Pull request by JonnyHayes**
Adds classnames to insant search for GTM props.

### CODE REVIEW

**Comment by tw2113** [comment]
Related to this forum thread https://wordpress.org/support/topic/extending-the-search-2/

File: templates/instantsearch.php [hunk 1]
```diff
@@ -33,18 +33,18 @@
...
 			<# } #>
 
 			<div class="ais-hits--content">
-				<h2 itemprop="name headline"><a href="{{ data.permalink }}" title="{{ data.post_title }}" itemprop="url">{{{ data._highlightResult.post_title.value }}}</a></h2>
+				<h2 itemprop="name headline"><a href="{{ data.permalink }}" title="{{ data.post_title }}" class="ais-hits--title-link" itemprop="url">{{{ data._highlightResult.post_title.value }}}</a></h2>
 				<div class="excerpt">
 					<p>
 			<# if ( data._snippetResult['content'] ) { #>
-			  <span class="suggestion-post-content">{{{ data._snippetResult['content'].value }}}</span>
+			  <span class="suggestion-post-content" class="ais-hits--content-snippet">{{{ data._snippetResult['content'].value }}}</span>
```
Comment (richaber) [inline templates/instantsearch.php:47]: I assume this should be  
`class="suggestion-post-content ais-hits--content-snippet"`

and not  
`class="suggestion-post-content" class="ais-hits--content-snippet"`

Comment (JonnyHayes) [reply templates/instantsearch.php]: Yes you are correct.

**Review by MyAudioDNA — APPROVED** [review]
I don't see any issues with this.

**Review by richaber — APPROVED** [review]

**Pull request merged**